articles/fire-forests-inventories/components/rect.js

```diff
@@ -0,0 +1,35 @@
+import React, { memo } from 'react'
```

Comment:
Is it possible to use the `Rect` from `charts` instead here?

Reply:
yep! works great, forgot we did that